packages/fiori-elements-writer/templates/v2/common/add/webapp/test/changes_loader.js

```diff
@@ -9,7 +9,12 @@ const connectorPath =
         : 'sap/ui/fl/initial/api/connectors/FileListBaseConnector';
 
 sap.ui.define(['sap/base/util/merge', connectorPath], function(merge, FileListBaseConnector) {
-    var trustedHosts = [/^localhost$/, /^.*.applicationstudio.cloud.sap$/];
+    var trustedHosts = [
+        /^localhost$/,
```

Comment:
I feel it is very fragile to have the same content tested 4 times when a tiny mis-edit could cause a major regex issue. 
It would be good to test the contents in a test against a single known good output.

Reply:
Have add `packages/fiori-elements-writer/test/unit/templates.test.ts` let me know if that's in line with what you were thinking